blenderproc/scripts/download_cc_textures.py

```diff
@@ -19,49 +23,87 @@ def cli():
         'User-Agent': 'Mozilla/5.0'
     }
 
-    cc_texture_dir = args.output_dir
-    if not os.path.exists(cc_texture_dir):
+    cc_texture_dir = Path(args.output_dir)
+    if not cc_texture_dir.exists():
         os.makedirs(cc_texture_dir)
-    else:
-        raise Exception("The folder already exists!")
 
-    # download the csv file, which contains all the download links
-    csv_url = "https://cc0textures.com/api/v1/downloads_csv"
-    csv_file_path = os.path.join(cc_texture_dir, "full_info.csv")
-    request = requests.get(csv_url, headers=headers)
-    with open(csv_file_path, "wb") as file:
-        file.write(request.content)
-
-    # extract the download links with the asset name
+    # until all download files have been found
+    # this loop is necessary as the server only allows downloading the info for 100 materials at once
+    current_offset = 0
     data = {}
-    with open(csv_file_path, "r") as csv_file:
-        csv_reader = csv.DictReader(csv_file, delimiter=',')
-        for line in csv_reader:
-            if line["Filetype"] == "zip" and line["DownloadAttribute"] == "2K-JPG":
-                data[line["AssetID"]] = line["PrettyDownloadLink"]
-
+    while True:
+        offset_size = 100
+        # download the json file, which contains all information
+        json_url = f"https://ambientcg.com/api/v2/full_json?include=downloadData&limit={offset_size}" \
+                   f"&offset={current_offset}&type=material"
+        json_file_path = cc_texture_dir / f"full_info_{current_offset}.json"
+        request = requests.get(json_url, headers=headers)
+        with json_file_path.open("wb") as file:
+            file.write(request.content)
+        with json_file_path.open("r") as file:
+            json_data = json.load(file)
```

Comment:
Is there a reason why we cannot use the json data directly instead of writing it to a file first?
```suggestion
        request = requests.get(json_url, headers=headers)
        json_data = request.json()
```